demos/tests/args.py

```diff
@@ -40,6 +40,26 @@ def brats_arg(id):
 def image_retrieval_arg(id):
     return TestDataArg('Image_Retrieval/{}'.format(id))
 
+class NumThreads:
```

Comment:
What is the purpose of such classes?
Could we specify `single_option_cases('-nthreads', 1, 2, 4, 8),` or not?